docker-compose-prod.yml

```diff
@@ -0,0 +1,20 @@
+version: "3.8"
+
+services:
+  argos_file_manager_prod:
+    build:
+      context: .
+      dockerfile: Dockerfile.prod
+    image: argos-file-manager-prod
+    container_name: argos_file_manager_prod
+    ports:
+      - "8282:8282"
+    profiles:
+      - production
+    environment:
+      - AWS_REGION=${PROD_AWS_REGION}
+      - AWS_ACCESS_KEY_ID=${PROD_AWS_ACCESS_KEY_ID}
+      - AWS_SECRET_ACCESS_KEY=${PROD_AWS_SECRET_ACCESS_KEY}
+      - AWS_BUCKET_NAME=${PROD_AWS_BUCKET_NAME}
+    env_file:
+      - .env
```

Comment:
when running docker compose, it compiles correctly but nothing is displayed on the port where the project should run.
![image](https://github.com/user-attachments/assets/a67670cc-094d-4e91-bbf9-d9750cf0728c)
![image](https://github.com/user-attachments/assets/b67b8b04-409b-4c61-9756-2c27254d355d)
![image](https://github.com/user-attachments/assets/b46229cc-8800-48da-b647-a485b40c60c7)

Reply:
yeah, I was setting an incorrect port, the problem now is solved, thanks